pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,48 @@
+{ 
+  pkgs
```

Comment:
This is not used.

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/f70af1e46c6f7fa18556a83889bde4547972551a